### ISSUE TITLE
Proposal - new chapter 16 on Grey Literature

### ISSUE THREAD

**Issue by tosteiner** [OpenScienceMOOC/Module-6-Open-Access-to-Research-Papers#24]
Grey lit might be understood as the backbone of Open Access, with the term referring 

>>>

> to  “any  documen-tary  material  that  is  not  commercially  published  and  is  typically  composed  of  technical  reports,  working  papers,  business  documents,  and  conference  proceed-ings”1  or  the  “quasi-printed  reports,  unpublished  but  circulated  papers,  unpub-lished  proceedings  of  conferences,  printed  programs  from  conferences,  and  the  other  non-unique  material  which  seems  to  constitute  the  bulk  of  our  modern [..] manuscript collections.”2 In the educational context it could also include recorded lectures  and  other  course  content,  student  papers,  thesis’  repositories,  etc.  The  dominant theme of these conceptualizations is the unpublished nature of the literature [...]
> >>>
(Lipinski, in Farace, Schöpfel 2010. p.85f. https://hal.archives-ouvertes.fr/hal-01981296/document )

How about we dedicate a chapter to this, then including #22

**Comment by tosteiner** [comment]
links

* https://en.wikipedia.org/wiki/Grey_literature
* https://libguides.tulane.edu/openaccess/greylit
* https://guides.dml.georgetown.edu/greylit/resources
* https://onlinelibrary.london.ac.uk/resources/databases/opengrey
* https://guides.lib.umich.edu/c.php?g=283076&p=1886024

**Comment by Protohedgehog** [comment]
Hmmm. Do you think it is that important? I can just mention Repec and some other repos in the existing structure.

**Comment by tosteiner** [comment]
yeah, sure - won't be able to whip out a whole chapter in time for release, maybe somewhen in the longer run?

**Comment by Protohedgehog** [comment]
Yep, totes. Let's keep it in mind as something for the future.
------------------------------------------------------------------------------

*Rogue Palaeontologist; PhD, MEarthSci, MSc *

*Research Fellow: Center for Research and Interdisciplinarity, Paris;
Southern Denmark University, Odense*

*Latest research papers*: Ten hot topics around scholarly publishing
<https://www.mdpi.com/2304-6775/7/2/34>; Open Access in Palaeontology
<https://palaeo-electronica.org/content/2019/2548-open-access-in-palaeontology>;
Ten simple rules for researchers collaborating on Massively Open Online
Papers (MOOPs) <https://osf.io/preprints/metaarxiv/et8ak>.

   - Founder of the *Open Science MOOC* <https://opensciencemooc.eu/> (open
   Slack <https://osmooc.herokuapp.com/> channel)



   - Founder of *paleorXiv* <https://paleorxiv.org/>, a free, open source
   digital publishing platform for all Palaeontology research (companion
   Website) <https://paleorxiv.github.io/>
   - Independent open science communicator and consultant
   <http://fossilsandshit.com/open-scholarship/>
   - Author of Excavate! Dinosaurs
      <http://www.amazon.co.uk/Excavate-Dinosaurs-Paper-Toy-Palaeontology/dp/178240144X>
and
      World of Dinosaurs (coming 2018)

*Personal website <http://fossilsandshit.com/> - Home of the Green Tea and
Velociraptors blog.*

*ORCID:* 0000-0001-7794-0218 <http://orcid.org/0000-0001-7794-0218>
*Twitter:* @protohedgehog
<https://twitter.com/Protohedgehog>
